app/api/custom/attendees.py

```diff
@@ -45,3 +48,33 @@ def send_receipt():
             return jsonify(message="receipt sent to attendees")
     else:
         raise UnprocessableEntityError({'source': ''}, 'Order identifier missing')
+
+
+@attendee_blueprint.route(
+    '/events/<int:event_id>/attendees/<int:attendee_id>/state', methods=['GET']
```

Comment:
how about: /v1/states?event=<int:event_id>&attendee=<int: attendee_id>

Reply:
route is updated